src/index.js

```diff
@@ -472,6 +472,7 @@ function createConfig(options, entry, format, writeMeta) {
 			: () => resolve(options.cwd, 'mangle.json');
 
 	const useTypescript = extname(entry) === '.ts' || extname(entry) === '.tsx';
+	const emitDeclaration = !!options.generateTypes;
```

Comment:
maybe just checking if `pkg.types || pkg.typings` is present in package.json would be enough to trigger this?

Reply:
Love that idea! Made https://github.com/developit/microbundle/pull/631 that does enable declarations only when `pkg.types || pkg.typings` is present 🎉

Reply:
@marvinhagemeister I copied your change into this PR 👍